src/EditorFeatures/CSharpTest/InlineDeclaration/CSharpInlineDeclarationTests.cs

```diff
@@ -71,19 +71,23 @@ void M()
         public async Task InlineVariableWithConstructor1()
         {
             await TestInRegularAndScriptAsync(
-@"class C
+@"class C1
```

Comment:
Had to fix this test to be a valid program, otherwise flow analysis gives false positive.